python/cudaq/__init__.py

```diff
@@ -87,6 +87,26 @@ def synthesize(kernel, *args):
                              kernelName=kernel.name)
 
 
+def simulation_dtype():
+    """
+    Return the data type for the current simulation backend, 
+    either `complex128` or `complex64`.
```

Comment:
Are you really supporting quad precision? I suspect this is meant as `complex_fp64` and `complex_fp32` though. (Note that there are also things like complex<int> out there, so using the `fp` won't be a bad idea.

Reply:
correct me if I'm wrong, but `complex128 == 2 * doubles (FP64)` and `complex64 == 2 floats (FP32)` right? Numpy seems to use this jargon for `complex_on_fp64` and `complex_on_fp32`